handbook/network-security/network-security.md

```diff
@@ -0,0 +1,225 @@
+
+- **File Ingestion Service Storage**
+    - The File Ingestion Services (FIS) can be configured to store files on either Network Attached Storage (NAS) or AWS S3.
+    - NAS uses the SMB protocol.
+        - The SMB protocol can be configured to use AES encryption. Refer to [SMB security enhancements](https://docs.microsoft.com/en-us/windows-server/storage/file-server/smb-security) for details and steps to secure this transport.
+    - AWS S3 uses HTTPS.
+        Refer to the [SystemLink manual](https://www.ni.com/documentation/en/systemlink/latest/data/uploading-files-to-amazon-s3/) for steps to enable S3 file storage.
+
+- **Data Finder**
+    - DataFinder enables indexing and searching for files stored on network drives.
+    - DataFinder's network communication cannot be secured with TLS.
+
+- **Identity Provider**
+    - Active Directory, LDAP, and OpenID Connect are the supported identity providers.
+        - Local Windows account may also be used (not shown)
+        - Active Directory used the Active Directory protocol.
+        - LDAP uses the `ldap` or the `ldaps` protocols.
+        - OpenID Connect uses HTTP or HTTPS. Refer to your OpenID Connect Provider's documentation on steps to enable HTTPS communication.
+    - Refer to [Sign-on with LDAP](/ldap/ldap/). and [Single Sign-on with OpenID Connect](/openid-connect/openid-connect/) for details on setting up these identity providers.
+
+- **NI VLM**
+    - NI Volume License Manager (VLM) is used to enforce SystemLink node licenses.
+    - NI VLM uses TCP and does not support encrypted communication.
+    - Refer to [Licensing SystemLink Products](https://www.ni.com/documentation/en/systemlink/2020r4/setup/licensing-systemlink/) for details on using VLM with SystemLink.
+
+## NI Web Server
+
+NI Web Server is based on [**Apache httpd**](https://httpd.apache.org) and includes the **NI Web Server Configuration** utility to aid users in choosing secure presets as well as a GUI for changing Apache httpd configurations.
+
+!!! note
+    NI Web Server ships with SystemLink and various other NI software products such as LabVIEW.
+
+    Refer to the [NI Web Server manual](https://www.ni.com/documentation/en/ni-web-server/latest/manual/manual-overview/) for step by step instructions to configure your web server.
+
+### HTTPS in NI Web Server
+
+NI Web Server supports TLS 1.2.
+
+**NI Web Server Configuration** supports creating self-signed certificates, certificate signing requests (CSR), and installing certificates generated by a certificate authority. The DNS settings for the server can affect the operability of these certificates. Refer to [**DNS Configuration**](#dns-configuration) for details.
+
+!!! warn
+    Self signed certificates should be used for testing purposes only.
+
+!!! note "Limited capabilities when using self-signed certificates"
+
+    When using a self-signed certificates, clients will not trust the  by default. This can prevent some operations from occurring.
```

Comment:
Missing a word after "trust the", maybe certificates? Or the -> them?  What about:
```suggestion
   Clients will not trust self-signed certificates by default. This can prevent some operations from occurring.
```